projects/Rockchip/devices/OdroidGoAdvance/packages/python-evdev/package.mk

```diff
@@ -0,0 +1,47 @@
+PKG_NAME="python-evdev"
```

Comment:
If this is a generic package, please place it in a generic place (not in OGA directories, right?)

Reply:
this package only needed for oga_events.py